src/fs/copy.js

```diff
@@ -1,5 +1,20 @@
+import fs from 'fs/promises';
+import { existsSync } from 'fs';
+
 const copy = async () => {
-    // Write your code here 
+    const src = new URL('files', import.meta.url);
+    const dst = new URL('files_copy', import.meta.url);
+    try {
+        if (!existsSync(src)) {
+            throw null;
+        }
+        if (existsSync(dst)) {
+            throw null;
+        }
+        await fs.cp(src, dst, { recursive: true });
```

Comment:
It is better to avoid multiple file system operations for the following reasons:
    - it's damn slow
    - if OS-Scheduler interrupts a process after `existsSync` and before an `fs` operation and give control to another process that deletes the file, then the file will not exist when `fs` is executed. **Prefer to use NodeJS flags.**

You should follow the rule: `one operation for one file.`